src/components/form/examples/dynamic-form.tsx

```diff
@@ -45,7 +45,14 @@ export class DynamicFormExample {
 
     public render() {
         return [
-            <textarea onChange={this.handleTextChange}>{this.text}</textarea>,
+            <limel-code-editor
+                language="json"
+                lineNumbers={true}
+                lint={true}
+                fold={true}
+                onChange={this.handleTextChange}
+                value={this.text}
+            ></limel-code-editor>,
```

Comment:
```suggestion
            />,
```

Just a style/taste. But the code looks cleaner and we normally do that when elements don't wrap anything else.